Examples/Framework/src/Framework/Sequencer.cpp

```diff
@@ -290,21 +291,25 @@ int ActsExamples::Sequencer::run() {
                 throw std::runtime_error("Failed to decorate event context");
               }
             }
-            // Read everything in
+
+            ACTS_VERBOSE("Execute readers");
             for (auto& rdr : m_readers) {
               StopWatch sw(localClocksAlgorithms[ialgo++]);
               if (rdr->read(++context) != ProcessCode::SUCCESS) {
                 throw std::runtime_error("Failed to read input data");
               }
             }
-            // Execute all algorithms
+
+            ACTS_VERBOSE("Execute algorithms");
             for (auto& alg : m_algorithms) {
               StopWatch sw(localClocksAlgorithms[ialgo++]);
+              ACTS_VERBOSE("Execute Algorithm: " << alg->name());
```

Comment:
It feels somewhat strange that algorithm names get spelled out, but not reader/writer names. May want to add those as well?

Reply:
I can add that no problem.

Reply:
Added verbose output for context decorators, readers and writers.